docs/getting-started.md

```diff
@@ -35,7 +35,7 @@ git clone --recurse-submodules https://github.com/oss-review-toolkit/ort.git
 To build the command line interface run:
```

Comment:
Commit message nit: Because of the repeated "change" I had to read the message multiple times before I understood it. Maybe "change the directory to cd to" or "change the name of the root directory"?

Reply:
I've changed the wording in yet another way. Hope you like it 😁